packages/@aws-cdk/aws-codepipeline/README.md

```diff
@@ -141,11 +143,11 @@ You can also add the Action to the Pipeline directly:
 
 ```ts
 // equivalent to the code above:
-const buildAction = jenkinsProvider.addToPipeline(buildStage, 'JenkinsBuild', {
+const buildAction = jenkinsProvider.asCodePipelineAction('JenkinsBuild', {
   projectName: 'MyProject',
 });
 
-const testAction = jenkinsProvider.addToPipelineAsTest(buildStage, 'JenkinsTest', {
+const testAction = jenkinsProvider.asCodePipelineTestAction('JenkinsTest', {
```

Comment:
rename `toCodePipelineAction` with an option of `test: true`

Reply:
@skinny85 and I talked this over today. What do you think of deferring this change to a follow up pull request? The challenge at the moment with combining `asCodePipelineBuildAction` and `asCodePipelineTestAction` to `toCodePipelineAction` is the return types are different. I'd like to make `Action` a concrete type instead of `BuildAction`, `TestAction`, etc., but this seemed like a large change to include in an already large PR.

Reply:
Yep, we had this conversation originally when introducing these methods on CodeBuild Projects.

We need to return a different type from `asCodePipelineBuildAction` and `asCodePipelineTestAction`, and JSII does not support generics, hence the need for having 2 separate methods for these.